### PR TITLE
Tests for XmlAttributeCollection

### PR DESCRIPTION
**Pull request by plabovich**
Improves code coverage of XmlAttributeCollection from 47% to 92%

Contributes to #937

### CODE REVIEW

File: src/System.Xml.XmlDocument/tests/XmlAttributeCollectionTests/InsertAfterTests.cs [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            XmlDocument doc = CreateDocumentWithElement();
+            XmlElement element = doc.DocumentElement;
+            XmlAttribute refAttr = element.Attributes.Append(doc.CreateAttribute("attr1", "some:uri1"));
+            XmlAttribute newAttr = doc.CreateAttribute("attr2", "some:uri2");
+
+            XmlAttributeCollection target = element.Attributes;
+            Assert.Same(newAttr, target.InsertAfter(newAttr, refAttr));
+        }
+    }
+}
```
Comment (stephentoub) [inline src/System.Xml.XmlDocument/tests/XmlAttributeCollectionTests/InsertAfterTests.cs]: Nit, some newlines missing at ends of files.

File: src/System.Xml.XmlDocument/tests/XmlAttributeCollectionTests/RemoveAtTests.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        private XmlDocument CreateDocumentWithElement()
+        {
+            var doc = new XmlDocument();
+            doc.AppendChild(doc.CreateElement("root"));
+            return doc;
+        }
+
+        [Fact]
+        public void RemoteAtWithNegativeIndexReturnsNull()
```
Comment (stephentoub) [inline src/System.Xml.XmlDocument/tests/XmlAttributeCollectionTests/RemoveAtTests.cs]: Some of these test names have a typo: "Remote" instead of "Remove"

**Comment by stephentoub** [comment]
@plabovich, thanks for adding these!  A few minor comments, but otherwise they LGTM.

@krwq, any thoughts?

**Comment by plabovich** [comment]
@stephentoub, thanks for taking a look. I've corrected based on the feedback. Btw, regarding new lines at end of files, why does they matter?

**Comment by stephentoub** [comment]
> Btw, regarding new lines at end of files, why do they matter?

http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline

**Comment by stephentoub** [comment]
Thanks for the fixes. LGTM.

**Pull request merged**